src/esp32/ld/esp32.rom.libc-funcs.ld

```diff
@@ -0,0 +1,74 @@
+/* These are the newlib functions present in ESP32 ROM.
+   They should not be used when compiling with PSRAM cache workaround enabled.
+   See also esp32.rom.newlib-data.ld for the list of .data/.bss symbols
```

Comment:
@erhankur @antmak  Looks like for esp32s2 we also need [esp32.rom.newlib-data.ld](https://github.com/espressif/esp-idf/blob/aaebc374676621980878789c49d239232ea714c5/components/esp_rom/esp32/ld/esp32.rom.newlib-data.ld#L4)
with symbols used by these functions. If functions used by stub does not need these symbols so we need to update these header comment.